File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/aderare-oioro-report-filter.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { JsonDateConverter } from "../../json-mapper";

@JsonObject
export class AderareOioroReportFilterModel {

	@JsonProperty("aniList", [Number])
    public aniList: number[] = null;
    
    @JsonProperty("organismIdList", [Number])
    public organismIdList: number[] = null;
    
    @JsonProperty("abreviere", String)
    public abreviere: string = null;
    
    @JsonProperty("comitetIdList", [Number])
    public comitetIdList: number[] = null;
    
    @JsonProperty("institutieIdList", [Number])
    public institutieIdList: number[] = null;
    
    @JsonProperty("reprezentantIdList", [Number])
    public reprezentantIdList: number[] = null;
    
    @JsonProperty("functie", String)
    public functie: string = null;
    
    @JsonProperty("coordonatorArbIdList", [Number])
	public coordonatorArbIdList: number[] = null;

}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/AderareOioroReportFilterModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.List;

public class AderareOioroReportFilterModel {

	private List<Integer> aniList;
	private List<Long> organismIdList;
	private String abreviere;
	private List<Long> comitetIdList;
	private List<Long> institutieIdList;
	private List<Long> reprezentantIdList;
	private String functie;
	private List<Long> coordonatorArbIdList;
	public List<Integer> getAniList() {
		return aniList;
	}
	public void setAniList(List<Integer> aniList) {
		this.aniList = aniList;
	}
	public List<Long> getOrganismIdList() {
		return organismIdList;
	}
	public void setOrganismIdList(List<Long> organismIdList) {
		this.organismIdList = organismIdList;
	}
	public String getAbreviere() {
		return abreviere;
	}
	public void setAbreviere(String abreviere) {
		this.abreviere = abreviere;
	}
	public List<Long> getComitetIdList() {
		return comitetIdList;
	}
	public void setComitetIdList(List<Long> comitetIdList) {
		this.comitetIdList = comitetIdList;
	}
	public List<Long> getInstitutieIdList() {
		return institutieIdList;
	}
	public void setInstitutieIdList(List<Long> institutieIdList) {
		this.institutieIdList = institutieIdList;
	}
	public List<Long> getReprezentantIdList() {
		return reprezentantIdList;
	}
	public void setReprezentantIdList(List<Long> reprezentantIdList) {
		this.reprezentantIdList = reprezentantIdList;
	}
	public String getFunctie() {
		return functie;
	}
	public void setFunctie(String functie) {
		this.functie = functie;
	}
	public List<Long> getCoordonatorArbIdList() {
		return coordonatorArbIdList;
	}
	public void setCoordonatorArbIdList(List<Long> coordonatorArbIdList) {
		this.coordonatorArbIdList = coordonatorArbIdList;
	}
		
}
